src/ui/public/notify/__tests__/notifier.js

```diff
@@ -185,6 +197,60 @@ describe('Notifier', function () {
     });
   });
 
+  describe('#custom', function () {
+    let customNotif;
```

Comment:
Am I being too pet-peevey to ask that we use unabbreviated variable names? In this case, `customNotification`. It makes it easier for me to scan and associate things when we use complete words instead of abbreviations, but it might just be me.